tests/rados/test_online_reads_balancer.py

```diff
@@ -0,0 +1,168 @@
+"""
+Module to test Reads balancer functionality on RHCS 8.0 and above clusters
+
+"""
+import re
+import time
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.rados.pool_workflows import PoolFunctions
+from utility.log import Log
+from utility.utils import method_should_succeed
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Module to test Reads balancer functionality on RHCS 8.0 and above clusters
+    Returns:
+        1 -> Fail, 0 -> Pass
+    """
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    rados_obj = RadosOrchestrator(node=cephadm)
+    pool_obj = PoolFunctions(node=cephadm)
+
+    regex = r"\s*(\d.\d)-rhel-\d"
+    build = (re.search(regex, config.get("build", config.get("rhbuild")))).groups()[0]
+    if not float(build) >= 8.0:
+        log.info(
+            "Test running on version less than 7.0, skipping verifying Reads Balancer functionality"
```

Comment:
Update comment

Reply:
Given that the test is being included in test suites Squid and above, do we really need this block of code?